ERCS/erc-7761.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: 7761
```

Comment:
```suggestion
eip: 7770
```
Assigning next sequential EIP/ERC/RIP number.  Numbers are assigned by editors & associates, please don't assign your own number.

Please also update the filename.

Reply:
done. fyi, the CI bot kind of forced me to assign a number, which contradicts the instructions in the template.